zhaquirks/owon/Owon_PC321_Z.py

```diff
@@ -0,0 +1,99 @@
+"""QUIRK FOR OWON PC321 Z"""
+
+import logging
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, Identify
+from zigpy.zcl.clusters.smartenergy import Metering
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
I think this (and the `import logging` import)  can be removed.

Reply:
I removed it